### ISSUE TITLE
Request: Add CLI Completion

### ISSUE THREAD

**Issue by willwill2will54** [framesurge/perseus#325]
This issue is requesting an enhancement to Perseus. Details of the scope will be available in issue labels.
The user described the problem related to this request as follows:
> Perseus-cli is missing CLI Completion

The user described the issue as follows:
> I would like to request the adding of CLI completion definitions for the perseus-cli tool. Perseus-cli uses clap under the hood, which has functionality for generating completions at run-time or compile-time. Which to do would require discussion, as each has trade-offs.
- The author is willing to attempt an implementation: true 

<details>
<summary>Tribble internal data</summary>

dHJpYmJsZS1yZXBvcnRlZCxDLWVuaGFuY2VtZW50LGF1dGhvci13aWxsaW5nLXRvLWltcGw=

</details>

**Comment by arctic-hen7** [comment]
Yes I've been meaning to do this for a while but just haven't gotten around to it! I think compile-time completions sound good unless you see any particular problem with that? I'd be very happy to accept a PR for this.
